src/main/java/com/epam/reportportal/cucumber/Utils.java

```diff
@@ -39,59 +39,68 @@
 import rp.com.google.common.collect.ImmutableMap;
 
 import java.util.Calendar;
+import java.util.Date;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
 
-public class Utils {
```

Comment:
i believe utils are public by some reason. Someone might be using it in custom integrations

Reply:
changed. reverted back